### PR TITLE
testing and integration

### PR DESCRIPTION
**Pull request by Perfect111**
download networked aframe and configure socket io and easyRTC

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,18 @@
   "version": "0.2.0",
   "license": "MIT",
   "scripts": {
-    "start": "node src/server.js"
+    "start": "node src/server.js",
+    "dev": "node app.js"
```
Comment (dmitrizagidulin) [inline package.json:7]: @Perfect111 what is the difference between `server.js` and `app.js`?

**Comment by dmitrizagidulin** [comment]
I restored the accidentally deleted files (Dockerfile, etc). Proceeding to testing.

**Comment by shawticus** [comment]
server.js was the existing server from NAF, app.js is probably the easyrtc one I assume @Perfect111 is starting from for developing NAF Jitsi adapts

**Comment by dmitrizagidulin** [comment]
@shawticus Removed `src/server.js`, rebased. 

I don't know how to test this, so I'll leave it to you to merge.

**Pull request merged**